src/Upgrade/Infrastructure/VersionControlSystem/Generator/PullRequestDataGenerator.php

```diff
@@ -221,6 +219,18 @@ protected function buildReleaseGroupsTable(
         return $text;
     }
 
+    /**
+     * @param \ReleaseApp\Infrastructure\Shared\Dto\ReleaseGroupDto $releaseGroupDto
+     *
+     * @return string
+     */
+    protected function getReleaseGroupName(ReleaseGroupDto $releaseGroupDto): string
+    {
+        return mb_strlen($releaseGroupDto->getName()) > static::RELEASE_GROUP_TITLE_LENGTH
```

Comment:
it's wrong behavior. The name contains task id. it's not public data.
Internal Prs (demoshops) should have  full title with Task name id, but the customer should have only name as public RG page.  I'll describe detail the QA task.

Reply:
+